src/main/java/com/onarandombox/MultiverseCore/commandtools/MVCommandCompletions.java

```diff
@@ -66,4 +68,16 @@ private Collection<String> suggestMVWorlds(BukkitCommandCompletionContext contex
 
         return worlds;
     }
+
+    private Collection<String> suggestGamerules(BukkitCommandCompletionContext context) {
+
+        // Changes Collection of GameRules to Collection of Strings
+        Collection<String> gameRules = new ArrayList<>(Collections.emptyList());
```

Comment:
```suggestion
        Collection<String> gameRules = new ArrayList<>();
```
There isnt a need for `Collections.emptyList()`, ArrayList already implements the Collection interface